doc/source/learning/airship.rst

```diff
@@ -0,0 +1,100 @@
+Airship
+=====================
+
+Airship is a platform that takes YAML documents and turns them into Clouds.
+Takes helm charts, baremetal, security policies and network description and creates a cloud
+
+.. image:: https://airship-treasuremap.readthedocs.io/en/latest/_images/architecture.png
+
+* Airship architecture
+    https://airship-treasuremap.readthedocs.io/en/latest/
+
+* github
+    https://github.com/airshipit
+
+* Video
+    https://www.youtube.com/watch?v=0eEisMm9ykg
+
+
+Armada
+++++++++++++++++
+
+Armada uses a YAML to describe a multi-chart deployment. The keys are:
+
+* Armada is going to communicate with helm to create the desired infrastructure.
+* The configuration can be stored on the Deckhand.
+* Shipyard is going to command Armada.
+
+* Documentation
+    https://airship-armada.readthedocs.io/
+
+* github
+    https://github.com/airshipit/armada
+
+* Guide
+    https://airship-armada.readthedocs.io/en/latest/operations/guide-use-armada.html
+
+* Video
+    https://www.youtube.com/watch?v=ae-g9irTsXY
+
+
+Deckhand
+++++++++++++++++
+
+Deckhand is a document-based configuration storage service using barbican
+
+* Documentation
+    https://airship-deckhand.readthedocs.io/en/latest/
+
+* github
+    https://github.com/airshipit/deckhand
+
+
+Shipyard
```

Comment:
If this is the entrypoint in airship 1.0, shouldn't we explain this first?

Reply:
I agree